frontend/src/app/menu/menu.component.ts

```diff
@@ -40,6 +41,14 @@ export class MenuComponent implements OnDestroy, OnInit {
         private route: ActivatedRoute
     ) {}
 
+    /**
+     * whether the brand should explicitly add "i-analyzer" to the app name; not
+     * necessary if the app name already includes "i-analyzer"
+     */
+    get showIAnalyzerInBrand() {
+        return ! this.appName.toLowerCase().includes('i-analyzer');
```

Comment:
I like the solution stylistically, but I don't know if this is the best way to implement naming conventions. I would either add some documentation explaining how the naming of apps works and how a developer should use those values, or just give two fields in the environment for Main app name and Secondary App name or something? Or is this meant to enforce that we have the I-analyzer name present at all times? :p